algeria/researcher_profile/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+
+
+class Institution(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+class Department(models.Model):
+    name = models.CharField(max_length=255)
+    
+    def __str__(self):
+        return self.name
+
+class Position(models.Model):
+    name = models.CharField(max_length=255)
+    
+    def __str__(self):
+        return self.name
+
+class Research_Area(models.Model): ####implement fuzzy matching / standardization of names
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Collaboration_Area(models.Model): ####implement fuzzy matching / standardization of names
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+class Link(models.Model):
+    link = models.URLField(max_length = 255, null=True, blank=True)
+    link_type = models.CharField(max_length=255)
+    link_type = models.CharField(max_length=255)
```

Comment:
There seems to be to fields named `link_type` -- seems like we can get rid of one of them?